txpipe/sysmaps.py

```diff
@@ -156,8 +161,9 @@ def run(self):
             print("Finalizing maps")
         depth_pix, depth_count, depth, depth_var = depth_mapper.finalize(self.comm)
         map_pix, ngals, g1, g2, var_g1, var_g2 = mapper.finalize(self.comm)
+        map_pix, ngals, g1, g2, var_g1, var_g2 = mapper_psf.finalize(self.comm)
+
 
-        
         # Only the root process saves the output
         if self.rank==0:
             print("Saving maps")
```

Comment:
(sure you know this, but for completeness - need to save the PSF map as well as the others)